Packs/XMCO/Integrations/Serenety/Serenety.py

```diff
@@ -0,0 +1,251 @@
+
+
+''' HELPER FUNCTIONS'''
+
+
+def convert_to_demisto_severity(severity: str) -> float:
+    """
+    Maps XMCO severity to Cortex XSOAR severity.
+    Converts the XMCO alert severity level ('none', 'low', 'medium', 'high', 'critical') to Cortex XSOAR alert
+    severity (1 to 4).
+
+    Args:
+        severity (str): severity as returned from the LePortail API.
+
+    Returns:
+        int: Cortex XSOAR Severity (0,5 to 4)
+    """
+    return {
+        'none': IncidentSeverity.INFO,
+        'low': IncidentSeverity.LOW,
+        'medium': IncidentSeverity.MEDIUM,
+        'high': IncidentSeverity.HIGH,
+        'critical': IncidentSeverity.CRITICAL
+    }.get(severity, IncidentSeverity.INFO)
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'User <username> is authenticated' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'User <first_name> <surname> is authenticated' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    try:
+        result = client.fetch_current_user()
+        if result.get('_error', None):
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            message = 'ok'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+
+    return message
+
+
+def fetch_incidents(client: Client,
+                    last_run: dict,
+                    first_fetch_time: str,
+                    scope: str = None, ) -> tuple[dict[str, str], list[dict]]:
+    """
+    This function will execute each interval (default is 1 minute).
+
+    Args:
+        client: XMCO LePortail client
+        last_run: A dict with a key containing the latest incident created time we got
+        from last fetch
+        first_fetch_time: If last_run is None then fetch all incidents since first_fetch_time
+        scope: filter incident by scope from LePortail
+
+    Returns:
+        next_run: This will be last_run in the next fetch-incidents
+        incidents: Incidents that will be created in Cortex XSOAR
+    """
+    # Get the last fetch time, if exists
+    last_fetch = last_run.get("last_fetch", None)
+    if last_fetch is None:
+        last_fetch = first_fetch_time
+    else:
+        last_fetch = datetime.now().strftime(ISO_8601_FORMAT)
+
+    incidents: list[dict[str, Any]] = []
+    items = client.fetch_alert(scope)
+    for item in items:
+        try:
+            data = item.get('data', {})
+            if data:
+                created = dateparser.parse(data['_created'])
+                incident = {
```

Comment:
Please add all the important fields to the mapper instead of mapping it through the feethc_incidents function.
It will be easier to modify for customers in the future.
@jlevypaloalto - any objections? I believe some fields should be mapped here like `rawJason` according to our BP.